airbyte-workers/src/main/java/io/airbyte/workers/process/DockerProcessBuilderFactory.java

```diff
@@ -80,4 +86,18 @@ private Path rebasePath(final Path jobRoot) {
     return DATA_MOUNT_DESTINATION.resolve(relativePath);
   }
 
+  private boolean checkImageExists(String imageName) {
+    final String[] split = imageName.split(":");
+    final String basename = split[0];
+    final String tag = split.length > 1 ? split[1] : "";
+
+    try {
+      final Process checkProcess = new ProcessBuilder("./image_exists.sh", basename, tag).start();
```

Comment:
why do you need to separate the name from the tag? Why not just take it as it is?

Reply:
`https://index.docker.io/v1/repositories/$1/tags/$2`. for the url it needs to be split. just a choice of doing it here or in the bash script. this will be moot if i use docker pull.